lib/God.js

```diff
@@ -362,25 +362,30 @@ God.prepare = function prepare(env, cb) {
     // multi fork depending on number of cpus
     var arr = [];
     var instance_id = 0;
+    var increment = 0;
 
     (function ex(i) {
       if (i <= 0) {
         if (cb) return cb(null, arr);
         return false;
       }
 
-      env.NODE_APP_INSTANCE = instance_id++;
-      env.vizion_running = false;
+      if (env.en && env.env.increment_foreach)
+        env[env.env.increment_foreach] = increment++;
```

Comment:
@vmarchaud I assume `increment_foreach` is a new pm2 option. Therefore, I'd like to propose `increment` instead.

Btw we need to add this to the schema and the docs too!